_tasks/sync-proposal-data.mjs

```diff
@@ -0,0 +1,334 @@
+/**
+ * @file Synchronizes local proposal data with upstream GitHub proposal data.
+ * @author Derek Lewis <DerekNonGeneric@inf.is>
+ */
+
+// -----------------------------------------------------------------------------
+// Requirements
+// -----------------------------------------------------------------------------
+
+import { GhFileImporter } from './util/gh-file-importer.mjs';
+import { UnicodeEscapes } from './util/common.mjs';
+import { createRequire } from 'module';
+import { ifError } from 'assert';
+import { mkdir, readFile, rmdir, writeFile } from 'fs/promises';
+import { EOL as newlineMarker } from 'os';
+import { sep as pathSep } from 'path';
+import { writeSync } from 'fs';
+import htmlStringify from 'rehype-stringify';
+import raw from 'rehype-raw';
+import remark2rehype from 'remark-rehype';
+import unified from 'unified';
+import visit from 'unist-util-visit';
+
+const require = createRequire(import.meta.url);
+const yaml = require('js-yaml');
+const markdown = require('remark-parse');
+const html = require('remark-html');
+const table2json = require('tabletojson').Tabletojson;
+const log = require('fancylog');
+const he = require('he');
+
+const DIR_DATA = './_data';
+const DIR_TEMP = './tmp';
+const URL_RAW_PRPSLS_README =
+  'https://raw.githubusercontent.com/tc39/proposals/master/README.md';
```

Comment:
```suggestion
  'https://raw.githubusercontent.com/tc39/proposals/HEAD/README.md';
```

Reply:
This is amazing! I obviously had no idea that this existed. 🤯🤯🤯